app/hooks/useBitcoinBalance.tsx

```diff
@@ -1,20 +1,23 @@
 import { useLedgerBitcoinWallet } from './useLedgerBitcoinWallet';
 import useSWR from 'swr/esm/use-swr';
-import { btcIntoCurVal } from '../utils/currency';
-import { BigNumber } from 'ethers';
+import { btcIntoCurVal, ZERO_BTC } from '../utils/currency';
 
 export default function useBitcoinBalance() {
   const btcWallet = useLedgerBitcoinWallet();
-  const { data: balance } = useSWR(
+  const { data: balance, error: error } = useSWR(
     '/balance/btc',
     () => btcWallet.getBalance(),
     {
-      refreshInterval: 10000,
-      errorRetryInterval: 500,
-      errorRetryCount: 10,
-      initialData: BigNumber.from(0)
+      refreshInterval: 10000
     }
   );
 
+  if (!balance || error) {
+    return {
+      ...ZERO_BTC,
```

Comment:
Could we not display any figure instead?

Reply:
We display a Spinner if `isLoading` is set, no matter what the value is. I set the value to zero.